### ISSUE TITLE
The latest version looks become less stable

### ISSUE THREAD

**Issue by silence19** [prabirshrestha/dwm-win32#66]
It seems easier to crash than before.
Not quite sure, but do have the feeling.

**Comment by adhamramy** [comment]
Seems like commit 35baa8f has made crashes more common

**Comment by adhamramy** [comment]
When debugging the following error is thrown, in this case it occurred when `Ctrl+W` in the Files App. Windows 10 22H2

Exception thrown at 0x00007FFC8131246E (vcruntime140.dll) in dwm-win32.exe: 0xC0000005: Access violation reading location 0x0000000000000000.

**Comment by goyalyashpal** [comment]
> any open programs disappear

yep, i faced this too... but i was too clueless to even be able to file as an issue. glad i accidently came to issues and read this.

**Comment by goyalyashpal** [comment]
* the issue most likely seems to occur when interacting with the windows explorer's things 
* so, the taskbar, action center, virtual desktops (aka task view in win 10) etc
* combined with some action of dwm.... (like say tags, floating windows, etc)

**Comment by nir9** [comment]
@goyalyashpal The general issue is that dwm-win32 freezes when a program becomes unresponsive?

**Comment by goyalyashpal** [comment]
no, but rather that all the opened apps disappear - often times they seem to get closed, and rarely they get minimized.

**Comment by nir9** [comment]
@goyalyashpal The open apps seem to disappear when dwm-win32 crashes, you can use winlister to show again the windows that disappeared.

I opened a PR that improves the stability of dwm-win32 by not managing windows that are not responding - #71

**Comment by adhamramy** [comment]
Would it be viable to save a list of open windows, and automatically show them again on dwm-win32 start?

> @goyalyashpal The open apps seem to disappear when dwm-win32 crashes, you can use winlister to show again the windows that disappeared.
> 
> Working on a PR that improves the stability of dwm-win32 by not managing windows that are not responding - #71

Would it be viable to save a list of managed windows and automatically show them again on dwm-win32 start? Then the crashes wouldn't be a huge inconvenience.
